configuration.yaml

```
@@ -868,12 +1025,23 @@
   kind: helm
   name: kong
   repository: https://charts.konghq.com
+- kind: git
+  name: koperator
+  repository: https://github.com/adobe/koperator
+  searchPaths:
+    - config/base/crds
+  versionPrefix: v
 - kind: git
   kustomizationPaths:
     - config/crd
   name: kserve
   repository: https://github.com/kserve/kserve
   versionPrefix: v
+- entries:
+    - kubeblocks
+  kind: helm
+  name: kubeblocks
+  repository: https://apecloud.github.io/helm-charts
```

```suggestion
- kind: git
  name: kubeblocks
  repository: https://github.com/apecloud/kubeblocks
  searchPaths:
    - deploy/helm/crds
  versionPrefix: v
```

I was unable to make the helm chart produce crds

---

yeah I think that matches https://kubeblocks.io/docs/preview/user_docs/overview/install-kubeblocks which mentions that you have to install CRDs manually without helm - thanks for checking!